synapse/storage/databases/main/event_federation.py

```diff
@@ -1139,15 +1169,15 @@ def _get_backfill_events(self, txn, room_id, seed_event_id_list, limit):
             limit,
         )
 
-        event_id_results = set()
+        event_id_results: Set[str] = set()
 
         # In a PriorityQueue, the lowest valued entries are retrieved first.
         # We're using depth as the priority in the queue and tie-break based on
         # stream_ordering. Depth is lowest at the oldest-in-time message and
         # highest and newest-in-time message. We add events to the queue with a
         # negative depth so that we process the newest-in-time messages first
         # going backwards in time. stream_ordering follows the same pattern.
-        queue = PriorityQueue()
+        queue = PriorityQueue()  # type: ignore[var-annotated]
```

Comment:
Reason `synapse/storage/databases/main/event_federation.py:1180: error: Need type annotation for "queue"  [var-annotated]`

Reply:
I think this is `queue: "PriorityQueue[Tuple[int, int, str, str]]"`?